utils/dict_helpers.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import "strconv"
+
+// GetString returns the dict[key] or an empty string if the entry doesn't exists
+func GetString(dict map[string]string, key string) string {
+	if v, ok := dict[key]; ok {
+		return v
+	}
+	return ""
+}
+
+// GetStringInterface returns the dict[key] or an empty string if the entry doesn't exists or the value is not a string
+func GetStringInterface(dict map[string]interface{}, key string) string {
+	if v, ok := dict[key]; ok {
+		vString, ok := v.(string)
+		if !ok {
+			return ""
+		}
+		return vString
+	}
+	return ""
+}
+
+// GetUint64 returns the dict[key] or an empty uint64 if the entry doesn't exists
+func GetUint64(dict map[string]string, key string) uint64 {
+	if v, ok := dict[key]; ok {
+		num, err := strconv.ParseUint(v, 10, 64)
+		if err == nil {
+			return num
+		}
+		return 0
+	}
+	return 0
+}
```

Comment:
Even if we don't directly use them, bubbling up errors from these methods may be useful